### ISSUE TITLE
Would you be happy for this code to be changed to a more open license such as MIT or GPL? Kind regards

### ISSUE THREAD

**Issue by Edwin10151** [bitwidth/blackjack#1]

**Comment by bitwidth** [comment]
Yeah sure. Do you want me to add a GPLv3 license to this project?
By the way, this project is a bit old and it could have been written in a better way, and there is definitely more room for improvement.
Feel free to contribute and send PRs if you wish to.
